### PR TITLE
Fix incorrect numpy minimum version for astropy

### PR DESCRIPTION
**Pull request by mwcraig**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mwcraig** [comment]
@conda-forge-admin, please rerender

**Comment by mwcraig** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,11 +38,12 @@ requirements:
     - cython
     - setuptools_scm
     - extension-helpers
-    - numpy >=1.16
+    - numpy 1.17.*  # [py<3.9]
```
Comment (isuruf) [inline recipe/meta.yaml:41]:
```suggestion
    - numpy 1.17.*  # [py<39]
```

Comment (mwcraig) [reply recipe/meta.yaml:41]: Thanks @isuruf

**Comment by mwcraig** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -38,11 +38,12 @@ requirements:
     - cython
     - setuptools_scm
     - extension-helpers
-    - numpy >=1.16
+    - numpy 1.17.*  # [py<39]
+    - numpy         # [py>=3.9]
```
Comment (isuruf) [inline recipe/meta.yaml:42]:
```suggestion
    - numpy         # [py>=39]
```

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by mwcraig** [comment]
Thanks for all of your help, @isuruf

**Pull request merged**